content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+description: >
+    What are Operators
+---
+
+## What is an Operator?
+
+Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extending the Kubernetes API](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) with CustomResourceDefinitions it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and managed them by looking for the desired state. For a better understanding, see the [Operator Pattern][operator-pattern].
+
+## What is the difference between a Controller and an Operator
+
+See that following the [Operator's pattern][operator-pattern]; you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provides a reconcile function responsible for synchronizing the resources until a desired state on the cluster. For a more practical explanation, see also [What’s in a controller?](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller).
```

Comment:
This doesn't exactly state the difference between a controller and an operator and instead points towards doc to explain what a controller is and how it can be built. Could you refactor this part or maybe change the question for this answer?
If we are having this question, we could provide a short explanation that operators have application specific controllers, and native Kubernetes controllers are just reconcilers. (Something on similar lines).

Reply:
+1

Reply:
+1